boefjes/boefjes/plugins/kat_withaname/normalize.py

```diff
@@ -0,0 +1,72 @@
+import json
+from collections.abc import Iterable
+
+import structlog
+
+from octopoes.models import OOI, Reference
+from octopoes.models.ooi.findings import Finding, KATFindingType
+
+logger = structlog.getLogger(__name__)
+
+
+def run(input_ooi: dict[str, str], raw: bytes) -> Iterable[OOI]:
+    """Normalize witha.name output."""
+    result = json.loads(raw)
+
+    if result["targets"] is None:
+        logger.info("No targets found in witha.name output")
+        return
+
+    input_ooi_reference = Reference.from_str(input_ooi["primary_key"])
+    found_targets = None
+
+    ooi_category = "IP" if input_ooi["primary_key"].startswith("IPAddress") else "Hostname"
+
+    if ooi_category == "IP":
+        logger.debug("Found IP address: %s", input_ooi["primary_key"])
+        found_targets = [
+            target for target in result["targets"] if target["ip"] == input_ooi_reference.tokenized.address
+        ]
+    elif ooi_category == "Hostname":
```

Comment:
It can only be one of these two literals, so `elif` seems redundant
```suggestion
    else:
```

Reply:
I did this in case more OOIs would be added, but I shouldn't plan like that. Thank you

Fixed in 881e0c5